students/myszko_pawel/lesson_01_basics/07_Last digit of integer.py

```diff
@@ -0,0 +1,8 @@
+# Given an integer number, print its last digit.
+
+# Read an integer:
+a = int(input())
+a = str(a)
+# Print a value:
+print(a[-1])
```

Comment:
Nie jest to najbardziej wydajne rozwiązanie, ale nie poprawiaj. Optymalne jest wykorzystanie modulo.